content/developer/reference/frontend/generic_components.rst

```diff
@@ -7,6 +7,22 @@ To make it easier, the Odoo javascript framework provides a suite of generic
 components that can be reused in some common situations, such as dropdowns,
 checkboxes or datepickers. This page explains how to use these generic components.
 
+Reference List
+==============
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Technical Name
+     - Short Description
+   * - :ref:`CheckBox <generic_components/checkbox>`
+     - a simple checkbox component with a label next to it
+   * - :ref:`Dropdown <generic_components/dropdown>`
+     - full featured dropdown
+
+.. _generic_components/checkbox:
```

Comment:
I see that some internal hyperlink refs are prefixed with `javascript/`, some with `frontend/`, and others don't have a prefix. It could be a good idea to pick one or the other and stick to it throughout the Frontend doc. My personal preference would go to `.. _javascript/...:` or `.. _frontend/...:` as it gives some context and prevents colliding with other refs with the same name (e.g. /javascript/qweb vs /backend/qweb).

Reply:
completely true. i think `frontend` would make more sense. i'll do a pr with that later. it's now on my task list